TPC/TPCrec/AliTPCtracker.cxx

```diff
@@ -942,8 +950,8 @@ void AliTPCtracker::ErrY2Z2(AliTPCseed* seed, const AliTPCclusterMI * cl, double
   // additional systematic error on the cluster
   double serry2=0,serrz2=0;
   AliTPCcalibDB::Instance()->GetTransform()->ErrY2Z2Syst(cl, tgp, seed->GetTgl(), serry2,serrz2);
-  erry2 += serry2;
-  errz2 += serrz2;
+  // erry2 += serry2;            // MI change -add  error due distortion fluctuation  later  after rescaling
+  // errz2 += serrz2;            // MI change -add  error due distortion fluctuation  later  after rescaling
```

Comment:
The ErrY2Z2 method is also called by the HLT in https://github.com/alisw/AliRoot/blob/4083c3d591b9bc964c897e33fc4c06d047882caa/GPU/GPUTracking/oldFiles/GPUTPCGMOfflineFitter.cxx#L205
@davidrohr , is this "oldFile" still relevant ?
If it is, then, @miranov25 you could leave ErrY2Z2 untouched and on the lines https://github.com/alisw/AliRoot/blob/4083c3d591b9bc964c897e33fc4c06d047882caa/TPC/TPCrec/AliTPCtracker.cxx#L314-L315 subtract back the syst error: ``erry2Res -= seed->GetErrorY2Syst()``